drivers/i2c/sparkfundev15316_driver.go

```diff
@@ -0,0 +1,485 @@
+package i2c
+
+import (
+	"fmt"
+	"math"
+	"time"
+)
+
+type Channel int
```

Comment:
Because all i2c types, vars, constants are in the same package it would be more future proof to differentiate it with a package prefix.

Reply:
...and this cause the failed CI build: "defaultAddress" is duplicated